packages/data-transformation/csv-to-metrics/Dockerfile

```
@@ -0,0 +1,47 @@
+ARG py_version=3.12.11
+
+FROM python:$py_version-slim-bullseye AS base
+
+RUN apt-get update \
```

Hadolint (2.12.0)

[error] Do not use apt-get dist-upgrade

(DL3005)